### PR TITLE
Only Run Get Kuryr Services Task when cluster is running Kuryr

### PR DESCRIPTION
**Pull request by iamemilio**
The uninstall.yml playbook hangs when trying uninstall kuryr if kuryr is not installed. Now it checks to see if the cluster is configured to use kuryr before trying to uninstall it.

### CODE REVIEW

File: roles/openshift_openstack/tasks/unprovision.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@
     all_namespaces: true
   register: svc_output
   ignore_errors: true
+  when:
+    - openshift_use_kuryr|default(false) == true
```
Comment (michaelgugino) [inline roles/openshift_openstack/tasks/unprovision.yml]: should be
```
openshift_use_kuryr | default(False) | bool
```
True is implied.  Always need to cast to bool, don't want to test == true.  Probably should clean up other occurrences of this.

Comment (iamemilio) [reply roles/openshift_openstack/tasks/unprovision.yml]: do you want me to update all instances of  ` default( boolean value ) == boolean value`
to  `default( boolean value ) | bool`

**Comment by michaelgugino** [comment]
@iamemilio yeah, let's update all of them.

**Comment by iamemilio** [comment]
all instances of the conditional have been updated

**Comment by michaelgugino** [comment]
bot, test pull request

**Comment by michaelgugino** [comment]
@iamemilio this looks good to me.  Can you update the commit message and title to remove WIP when you're ready.

**Comment by iamemilio** [comment]
Ready for review and merge
- [x] changed all checks of openshift_use_kuryr to use the conditional | d() | bool structure
- [x] checks openshift_use_kuryr variable before doing Get Kuryr Service task
- [x] merged latest changes from repo

**Comment by tzumainn** [comment]
Tests well, thanks for the PR!

**Pull request merged**